driver/mysql/mysql_test.go

```diff
@@ -29,6 +29,22 @@ func TestMigrate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	migrate(t, driverUrl)
+
+	if _, err := connection.Exec(`DROP TABLE IF EXISTS yolo, yolo1, ` + tableName); err != nil {
```

Comment:
Ok got it. Sorry, I completely forgot about this "yolo" table.
Could you just dry it a little (the drop table is duplicated), and we're ok to merge this.

Thanks!